### PR TITLE
Https locally

### PR DESCRIPTION
**Pull request by timgl**
## Changes

- add a `bin/start-https` command to setup params to run locally under https. See also https://github.com/PostHog/posthog.com/pull/108
-
-

## Checklist
- [ ] All querysets/queries filter by Team (if applicable)
- [ ] Backend tests (if applicable)
- [ ] Cypress E2E tests (if applicable)

### CODE REVIEW

File: frontend/src/scenes/actions/ActionEdit.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export function ActionEdit({ actionId, apiURL, onSave, user, isEditor, simmer, s
         id: actionId,
         apiURL,
         onSave: (action, createNew) => onSave(action, !actionId, createNew),
-        params: '?include_count=1' + (temporaryToken ? '&temporary_token=' + temporaryToken : ''),
+        temporaryToken,
```
Comment (EDsCODE) [inline frontend/src/scenes/actions/ActionEdit.js:14]: spillover

File: frontend/src/scenes/actions/actionEditLogic.js [hunk 2]
```diff
@@ -59,10 +59,11 @@ export const actionEditLogic = kea({
                 return data
             })
             try {
+                let token = props.temporaryToken ? '?temporary_token=' + props.temporaryToken : ''
                 if (action.id) {
-                    action = await api.update(props.apiURL + 'api/action/' + action.id + '/', action)
+                    action = await api.update(props.apiURL + 'api/action/' + action.id + '/' + token, action)
                 } else {
-                    action = await api.create(props.apiURL + 'api/action/', action)
+                    action = await api.create(props.apiURL + 'api/action/' + token, action)
```
Comment (EDsCODE) [inline frontend/src/scenes/actions/actionEditLogic.js:66]: spillover

**Review by EDsCODE — COMMENTED** [review]
the code lgtm. is there something I'm missing in order to run this? both `./bin/start` and `./bin/start-https` give errors: `You're accessing the development server over HTTPS, but it only supports HTTP`

File: frontend/src/scenes/actions/ActionEdit.js (hunk 1 above)
Comment (timgl) [reply frontend/src/scenes/actions/ActionEdit.js]: Fixed

**Comment by timgl** [comment]
You'll need to follow https://posthog.com/handbook/engineering/setup-ssl-locally (title update is being deployed...) to make ssl work locally

**Pull request merged**